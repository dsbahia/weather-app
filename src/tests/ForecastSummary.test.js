import React from "react";
import { render } from "@testing-library/react";
import ForecastSummary from "../components/ForecastSummary";

describe("ForecastSummary", () => {
  const validProps = {
    date: 1111111,
    description: "Test Description",
    icon: "testIcon",
    temperature: {
      min: 10,
      max: 20,
    },
  };
  it("renders forecast summary", () => {
    const { rendered } = render(
      <ForecastSummary
        date={validProps.date}
        description={validProps.description}
        icon={validProps.icon}
        temperature={validProps.temperature}
      />,
    );
    expect(rendered).toMatchSnapshot();
  });
});
