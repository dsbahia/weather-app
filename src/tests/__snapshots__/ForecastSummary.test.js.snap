// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastSummary renders forecast summary 1`] = `undefined`;
